_posts/choosing-first-language.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Que linguagem de programação devo escolher para o meu ninja aprender?"
+date: "2022-09-17"
```

Comment:
Is the date supposed to be this?

Reply:
Yes. I didn't want it to be in August and the 17th seems like a good date to have this up and running